lib/modules/pageUtils.js

```diff
@@ -32,7 +32,11 @@ export const createHash = str => {
 
 export const urlFromPage = (page, mergePage) => {
   const { queryParams, hashParams, url } = mergePage ? merge(page, mergePage) : page;
-  const queryString = Object.keys(queryParams).length ? `${createQuery(queryParams)}` : '';
-  const hashString = Object.keys(hashParams).length ? `${createHash(hashParams)}` : '';
+  const queryString = queryParams && Object.keys(queryParams).length
+    ? `${createQuery(queryParams)}`
```

Comment:
It's dying on the `Object.keys(queryParams).length` before it even gets there if `queryParams` is `undefined`. But that's a good point... @schwers? You know why it was done this way?